src/acl/external/time_quota/ext_time_quota_acl.cc

```diff
@@ -416,14 +406,11 @@ int main(int argc, char **argv)
 
     program_name = argv[0];
 
-    while ((opt = getopt(argc, argv, "dp:l:b:h")) != -1) {
+    while ((opt = getopt(argc, argv, "dp:b:h")) != -1) {
         switch (opt) {
         case 'd':
             tq_debug_enabled = true;
             break;
-        case 'l':
```

Comment:
This PR should update release notes, but, to save time, you may want to wait for @yadij opinion regarding this feature removal before doing so.

Reply:
I thought about it; given the fact that it is not a major feature
I considered just recording it to the ChangeLog when this lands.
I'll also defer to Amos' judgement if he feels this deserves go to
the release notes

Reply:
Sounds good. I am OK with whatever you two can agree on with regard to release notes inclusion.

Reply:
This is a UI feature, so IMO it should have some release notes mention. Does not have to be a large mention, just something for us to point at when people get surprised that their custom logs no longer work.

Reply:
added. Please re-review